airbyte-webapp/src/packages/cloud/lib/auth/GoogleAuthService.ts

```diff
@@ -15,35 +17,16 @@ import {
   updatePassword,
   updateEmail,
   AuthErrorCodes,
+  signInWithPopup,
+  GoogleAuthProvider,
+  GithubAuthProvider,
 } from "firebase/auth";
 
 import { Provider } from "config";
 import { FieldError } from "packages/cloud/lib/errors/FieldError";
 import { EmailLinkErrorCodes, ErrorCodes } from "packages/cloud/services/auth/types";
 
-interface AuthService {
```

Comment:
:information_source: We never exported this type, and implementing the interface in TypeScript does not even help not needing to respecify the types in the `GoogleAuthService` anyway. Thus this interface is not really fulfilling any purpose (besides needing to arbitrarily type the method signature twice) and I removed it.